modules/adding-ibm-z-lpar-agent.adoc

```diff
@@ -53,6 +53,8 @@ boot-artifacts
     └─ agent.s390x-rootfs.img
 ----
 
+. Rename the `boot-artifacts` inside the `generic.ins` to match the names of the `boot-artifacts` generated by the installation program.
```

Comment:
```suggestion
. Rename the `boot-artifacts` file present in the `generic.ins` parameter file to match the names of the `boot-artifacts` file generated by the installation program.
```

For objects in mono space/back ticks, we must provide names. For example:
- "the `xyz` pod"
-  "a `test-deploy` deployment"
-  "an `example-abc` object".
- See the guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

I learned from line no 58 that these are files.